internal/cmd/configuration/command.go

```diff
@@ -0,0 +1,74 @@
+package configuration
+
+import (
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/config"
+)
+
+type command struct {
+	*pcmd.CLICommand
+	config *config.Config
+}
+
+type configurationOut struct {
+	Name     string `human:"Name" serialized:"name"`
+	Value    string `human:"Value" serialized:"value"`
+	ReadOnly bool   `human:"Read-Only" serialized:"read_only"`
+}
+
+func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "configuration",
+		Aliases: []string{"config"},
+		Short:   "Manage CLI configuration fields.",
+	}
+
+	c := &command{
+		CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner),
+		config:     cfg,
+	}
+
+	cmd.AddCommand(c.newSetCommand())
+	cmd.AddCommand(c.newListCommand())
```

Comment:
will you consider adding a getter?

Reply:
I'm still split on whether or not we should. `list` already shows all the configuration fields in a format where it's clear to the user what the value of each field is. 

Especially if `list` is refactored to only display the `configurable` fields (i.e. the ones modifiable with set), I think `get` doesn't really provide any new functionality.